plugin/storage/cassandra/spanstore/writer_test.go

```diff
@@ -192,13 +194,20 @@ func TestSpanWriter(t *testing.T) {
 				// note: using matchOnce below because we only want one tag to be inserted
 				w.session.On("Query", stringMatcher(insertTag), matchOnce()).Return(tagsQuery)
 
-				w.session.On("Query", stringMatcher(serviceNameIndex), matchEverything()).Return(serviceNameQuery)
-				w.session.On("Query", stringMatcher(serviceOperationIndex), matchEverything()).Return(serviceOperationNameQuery)
+				w.session.On("Query",
+					stringMatcher(serviceNameIndex),
+					matchEverything()).Return(serviceNameQuery)
+				w.session.On("Query",
+					stringMatcher(serviceOperationIndex),
+					matchEverything()).Return(serviceOperationNameQuery)
```

Comment:
(!) while I raised the issue with long lines, I only meant it for new/changed code. It is a very bad practice to reformat the code in the same PR as the main code changes, creates a lot of unnecessary noise.